packages/webapi/services/chain.go

```diff
@@ -46,22 +47,34 @@ func NewChainService(
 	}
 }
 
-func (c *ChainService) ActivateChain(chainID isc.ChainID) error {
-	_, err := c.chainRecordRegistryProvider.ActivateChainRecord(chainID)
+func (c *ChainService) ActivateChain() error {
+	ch, err := c.GetChain()
```

Comment:
Be careful here, the chain activation is a bit delicate. Rather keep it how it is.